binding-properties/src/main/java/com/iluwatar/bindingproperties/App.java

```diff
@@ -0,0 +1,51 @@
+package com.iluwatar.bindingproperties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is a simple test case for binding properties pattern.
+ */

```

Comment:
Please update the comment since this is not the test class. The comment here needs to include a basic introduction to the pattern and a description of how this sample implements it.